core/src/main/java/com/datastrato/gravitino/connector/BaseCatalog.java

```diff
@@ -2,11 +2,12 @@
  * Copyright 2023 Datastrato Pvt Ltd.
  * This software is licensed under the Apache License version 2.
  */
-package com.datastrato.gravitino.catalog;
+package com.datastrato.gravitino.connector;
```

Comment:
What's the meaning of the word `connector`, I can't fully link `connector` to class under this package like `OperationProxy`, `TableOperation`

Reply:
What I referred to is from Spark, which put all the interfaces/classes that can be used by users into the `connector` package. So users can use all the classes under this package to build their own catalog.

Do you have better suggestions?